stdlib/array.gr

```diff
@@ -229,7 +234,12 @@ provide let concat = arrays => {
  *
  * @param array: The array to copy
  * @returns The new array containing the elements from the input
- *
+ * 
+ * @example
+ * let array1 = [> 9, 8, 7, 6, 5]
+ * let array2 = Array.copy(array1) 
+ * assert array1 == array2
```

Comment:
You would often use copy for copying the exact contents rather then a reference to that so it might be worth chaning this to:
```suggestion
* array1[0] = 2
* assert array1 == [> 2, 8, 7, 6, 5]
* assert array2 == [> 9, 8, 7, 6, 5]
``` 
This shows that when you modify `array1` its not being copied by reference but by value.
But wait for feedback from the core team before you apply that.

Reply:
Yeha I see what you mean there maybe someone from the core team has a better example my only thought on that is the reason to use `copy` over just doing `let a = b` is that it clones the it. and your example doesnt really show that very well. There might be a better example though that shows its a clone rather then just a copy.